src/components/SearchBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import '../styles/search.scss'
```

Comment:
Could you create a separate SearchBar.scss in the components folder that we will use later? We shouldn't actually need search.scss on _this_ component, I think.